docs/source-pytorch/common/remote_fs.rst

```diff
@@ -17,7 +17,7 @@ Working with different filesystems can be accomplished by appending a protocol l
     trainer = Trainer(default_root_dir="s3://my_bucket/data/")
     trainer.fit(model)
 
-You could pass custom paths to loggers for logging data.
+As for ``TensorBoardLogger``, by default, we use ``tensorboardX``. To use it with ``fsspec`` support, make sure you have ``tensorboard`` installed AND NO ``tensorflow`` installed in the current environment, as ``tensorboard`` will use ``tensorflow``'s GFile when detected, and it only uses fsspec on absence of ``tensorflow``.
```

Comment:
I'm ok with this specific explanation. 

Here is an alternative suggestion that might work better:
1. Here, explain in general terms: "For logging, remote filesystem support depends on the particular logger integration being used. Consult the documentation of the individual logger for more details (and here include a link to our logger page overview)."
2. In each logger, we declare whether it supports remote filesystems or not. For example, in TensorBoardLogger we would write: "This logger supports writing to remote fileystems (s3, gcs, etc.) if you have the tensorboard package installed". In others we write: "This logger does not support writing to remote FS".

Reply:
Yes, this way is better. Tensorboard supports remote filesystems, for sure, and I think I can add support for CSVLogger. However, I'm not sure if other loggers, for example, mlflow and wandb, support remote filesystems.